index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.typekit.net/fhu4nhd.css">
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+  <!--Toolbar-->
+  <header>
+    <p class="logo">My Logo</p>
```

Comment:
- I'd like to kindly suggest that you wrap your logo `My Logo` in an `<a>` tag as this is meant to be a cliackable link (`a` tag). Apparently, this is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- I think it will be better if you replaced the text "My logo" with something a bit more personal. I believe using your name will be just fine.  Or what do you think? 🤔